tests/test_assets.py

```diff
@@ -450,16 +457,41 @@ def test_lookup_symbol(self):
                 self.assertEqual(result.symbol, 'EXISTING')
                 self.assertEqual(result.sid, i)
 
-    @parameterized.expand(
-        build_lookup_generic_cases()
-    )
-    def test_lookup_generic(self, finder, symbols, reference_date, expected):
+    def test_lookup_symbol_from_multiple_valid(self):
```

Comment:
These rules were determined based on the particular idiosyncrasies of an upstream vendor.  It's not appropriate for us to be encoding those assumptions into the business logic of an open source project.  It should be the responsibility of the code that's authoring these files to ensure that any overlaps are resolved before writing data to the database.